DeezyMatch/candidateRanker.py

```diff
@@ -275,36 +300,45 @@ def candidate_ranker(input_file_path="default", query_scenario=None, candidate_s
             query_candidate_pd = pd.DataFrame(all_queries, columns=['s1'])
             query_candidate_pd['s2'] = all_candidates
             query_candidate_pd['s2_orig'] = all_candidates_orig
+
             query_candidate_pd['label'] = "False"
     
             # Compute cosine similarity
             cosine_sim = cosine_similarity(vecs_query[iq:(iq+1)].detach().cpu().numpy(), 
                                            vecs_candidates.detach().cpu().numpy()[orig_id_candis])
             cosine_dist = 1. - cosine_sim
     
-            if not pretrained_model_path in [False, None]:
-                all_preds = candidate_conf_calc(query_candidate_pd, 
-                                                model, 
-                                                train_vocab, 
-                                                dl_inputs, 
-                                                cutoffs=(id_1_neigh - id_0_neigh))
-                query_candidate_pd['dl_match'] = all_preds.detach().cpu().numpy()
-    
+            if use_predict == True:
+                if not pretrained_model_path in [False, None]:
```

Comment:
Maybe we can combine the two conditions:

`if use_predict and (not pretrained_model_path in [False, None]):`

and then:

```
else:
    query_candidate_pd['dl_match'] = [None]*len(query_candidate_pd)
```

Reply:
True, done: https://github.com/Living-with-machines/DeezyMatch/pull/121/commits/daa3ee6d214bcabd081d0e3cedb42b7387a78955